src/videojs-http-streaming.js

```diff
@@ -813,7 +830,16 @@ class VhsHandler extends Component {
        */
       systemBandwidth: {
         get() {
-          const invBandwidth = 1 / (this.bandwidth || 1);
+          let bandwidthEst = this.bandwidth;
+
+          // NetworkInfo.downlink maxes out at 10Mb/s. In the event that the player estimates a bandwidth
+          // greater than 10Mb, use the larger value to ensure that high quality streams are not
+          // accidentally filtered out
+          if (this.options_.useNetworkInformation) {
+            bandwidthEst = Math.max(bandwidthEst, this.effectiveBandwidthInBits_);
+          }
+
+          const invBandwidth = 1 / (bandwidthEst || 1);
```

Comment:
the invBandwidth can be 0? why not 1.0 / (bandwidthEst || 1)?

Reply:
Caught up with @jiangyurong609. The concern here was that the value would floor to zero, but js will maintain a floating point here. Resolving the comment.